Source/WebCore/platform/Scrollbar.h

```diff
@@ -196,3 +198,7 @@ class Scrollbar : public Widget {
 
 SPECIALIZE_TYPE_TRAITS_WIDGET(Scrollbar, isScrollbar())
 
+#define SPECIALIZE_TYPE_TRAITS_MAC_SCROLLBAR(ToValueTypeName, predicate) \
```

Comment:
The `MAC` should not exist in this macro name.

Reply:
Changed to SPECIALIZE_TYPE_TRAITS_SCROLLBAR_HOLDS_SCROLLER_IMP.